MAUI/Maps/shape.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section explains about shapes and how to apply colors to them based on specific values in the .NET MAUI maps.
 
+To learn more about our .NET MAUI Maps layers, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps layers, you can check the following video.